project1/CLI_strings/main.c

```diff
@@ -0,0 +1,35 @@
+#include "main.h"
+
+
+int main(int argc, char *argv[]) {
+
+	///*handling main arguments*/
+	if (argc != 3) {
```

Comment:
**nit**: Really nice error handling. Two things to consider though: 
1) The comment is not really necessary it's usually better to leave the code to explain itself (you can add non-doc comments when there is a design choice that might not be clear to the reader, or when there is some complicated algorithm involved, and you want to provide a reference).
2) To make this error handling even better, you could also print the number of arguments the user provided or even the exact arguments they have provided, e.g., 
`printf("Wrong number of arguments, expected 2, got %i\n", argc);`